### DISCUSSION TITLE
How to fully re-install Mastodon and keep database

### DISCUSSION THREAD

**Discussion by ThatOneCalculator** [mastodon/mastodon#17292] [Q&A]
Hey y'all. Long story short, I think I completely messed up my Mastodon `live` directory from using multiple forks, accidentally removing files, and more. I've tried to save it but it's a lost cause, I keep error 500ing. How can I safely delete the directory while backing up the database/media files, then reinstall? Possibly using docker this time.

**Comment by Gargron** [comment]
The database is not in the `live` directory. Just do your reset, and connect to the same database.

**Reply by ClearlyClaire** [reply]
Media files may be in the `live` directory, though, depending on how you set it up. Also back up `live/public/system`.

**Comment by TheEssem** [comment]
For the most part, all you need in the live directory are the media files and .env.production; the database is stored elsewhere.

This could potentially help: https://docs.joinmastodon.org/admin/migrating/

**Reply by ThatOneCalculator** [reply]
I kinda sorta corrupted the live directory and all the media is gone. Is there a way I can at least keep the user accounts?

**Reply by TheEssem** [reply]
The user accounts and posts are stored in the database.

**Reply by Oshuma** [reply]
Is there a way to do a reset on the media files? A lot of remote user avatars/headers are 404'ing after moving my `live/public/system` dir to a mounted drive. I've tried `tootctl accounts cull` and `tootctl accounts refresh --all`. I'm running a self-hosted, single user instance, so not worried about nuking local account media.
